packages/core/src/app/order/OrderSummarySubtotals.test.tsx

```diff
@@ -0,0 +1,107 @@
+import { render, screen } from '@testing-library/react';
+import React from 'react';
+
+import { mapFromPayments } from '../giftCertificate';
+
+import { getOrder } from './orders.mock';
+import OrderSummarySubtotals from './OrderSummarySubtotals';
+
+describe('OrderSummarySubtotals', () => {
+    it('renders component', () => {
+        const order = getOrder();
+
+        render(
+            <OrderSummarySubtotals
+                coupons={order.coupons}
+                discountAmount={order.discountAmount}
+                giftCertificates={order.payments && mapFromPayments(order.payments)}
+                handlingAmount={order.handlingCostTotal}
+                shippingAmount={order.shippingCostTotal}
+                subtotalAmount={order.orderAmount}
+                taxes={order.taxes}
+            />,
+        );
+
+        expect(screen.getByText('Tax')).toBeInTheDocument();
+        expect(screen.getByText('20% off each item')).toBeInTheDocument();
+        expect(screen.getByText('savebig2015')).toBeInTheDocument();
+        expect(screen.getByText('$5.00 off the shipping total')).toBeInTheDocument();
+        expect(screen.getByText('279F507D817E3E7')).toBeInTheDocument();
+        expect(screen.getByTestId('cart-subtotal')).toBeInTheDocument();
```

Comment:
I think it's best if you verify the value that a shopper would see rather than just detecting the presence of a `div` element, because the latter doesn't actually prove that the shopper could see the expected value.

Reply:
Thanks @epng for helping, after the store config is loaded, each amount is showing, and we could test the expected value.